newrelic/docs-website
# fix nav collapser quirk

this fixes the nav arrow animation and the button not bouncing with the rest of the page on scroll

## screenshvideos

### before
[before.webm](https://github.com/newrelic/docs-website/assets/14365008/2b613a98-7e75-4500-8686-2d7f32a2ead3)

### after
[after.webm](https://github.com/newrelic/docs-website/assets/14365008/3df954df-571b-4563-bc5d-59dcdf9a2f60)

## testing

- go to homepage
- scroll to bottom
- when page bouncy, nav collapser should bouncy too, whether nav is open or closed
- go to doc page
- repeat
